OptiFineDoc/assets/minecraft/optifine/lang/pt_br.lang

```diff
@@ -630,11 +630,11 @@ of.options.LAZY_CHUNK_LOADING.tooltip.7=corretamente. Disponível apenas em mund
 
 of.options.RENDER_REGIONS=Render. de regiões
 of.options.RENDER_REGIONS.tooltip.1=Renderização de regiões
-of.options.RENDER_REGIONS.tooltip.2= Não - (padrão) desativar renderização de regiões
-of.options.RENDER_REGIONS.tooltip.3= Sim - ativar renderização de regiões
-of.options.RENDER_REGIONS.tooltip.4=Renderiza terrenos distantes mais rapidamente.
-of.options.RENDER_REGIONS.tooltip.5=Ative o VBO para melhores resultados.
-of.options.RENDER_REGIONS.tooltip.6=Não recomendado com placas de vídeo integradas.
+of.options.RENDER_REGIONS.tooltip.2= Não - renderização padrão
+of.options.RENDER_REGIONS.tooltip.3= Sim - renderização mais rápida
```

Comment:
I was looking at the original text.
Wouldn't it be more correct to have it as:
"Sim - usar regiões de renderização."

Reply:
This is just me being pedantic by the way, I think it's odd to omit the original message.
Although both works.

Reply:
Also: https://github.com/sp614x/optifine/blob/8c18dd81c3cf6cea89793dafda24b5dd21f510a5/OptiFineDoc/assets/minecraft/optifine/lang/en_us.lang#L635

Reply:
Agradeço os comentários. Só há um problema: os enunciados precisam ser muito concisos, caso contrário eles pulam para fora da caixa de texto. Manter o alinhamento do texto no inglês já é difícil, fica ainda mais no PB.

Reply:
Todos os recursos disponíveis para encurtar o enunciado estão sendo usados, e.g., elipses, sujeitos nulos, abreviações, simplificação lexical. Não temos muita escolha a não ser fazer isso.